get-started/install-golang.md

```diff
@@ -1,2 +1,34 @@
 # Install Golang
 
+In order to build TiDB from source code, you need to have Go installed on your development environment.
+
+Most of TiDB developers are using Linux or MacOS, but if you are using Windows 10, you might want to have [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10) prepared for better experience, although TiDB should compile and run on Windows 10, it is not expected to be deployed on Windows, and you might have to figure out many troubles on your own (although you are welcomed to share them on our [forum](https://internals.tidb.io/)).
+
+
+TiDB currently used Go1.13 to compile, but we have plan to upgrade the compiler to Go1.16 recently. You can of course go to [Go's download page](https://golang.org/dl/) and choose the version 1.13 and follow the [install instruction](https://golang.org/doc/install) to get the compiler installed.
+
+However, since we might update the compiler version in the near future, if you are under Linux or MacOS you are strongly suggested to try [gvm](https://github.com/moovweb/gvm) which stands for Go Version Manager.
+
+To install gvm, use
```

Comment:
maybe separate this document into 2 sections can help users quickly set up their environment:

- TL;DR
- Detailed Explaination

Reply:
should this be a common practice for all pages? since this page is itself short, if page short like this needs TL;DR, I would guess we need TL;DR for all pages?

Reply:
No. I mean you need to be brief and concise in TL;DR, list all the commands to execute in this section

Reply:
not to write the agenda of this section in TL;DR